assignment3.txt

```diff
@@ -0,0 +1,10 @@
+Christopher Liu
+
+1. No, there is no input for the formatted string. If there were, you would have to assume that i was declared as a positive integer. If a float, the loop would not work as it takes integers, and if it is negative, it would last forever.
```

Comment:
Correct - the assumptions also need would be that i is initialized prior to the loop and greater than 0